src/main/java/org/folio/services/ledger/LedgerTotalsService.java

```diff
@@ -126,16 +126,23 @@ private LedgerFiscalYearTransactionsHolder buildHolderSkeleton(String fiscalYear
       .withInitialAllocation(HelperUtils.calculateTotals(budgets, Budget::getInitialAllocation))
       .withAwaitingPayment(HelperUtils.calculateTotals(budgets, Budget::getAwaitingPayment))
       .withEncumbered(HelperUtils.calculateTotals(budgets, Budget::getEncumbered))
-      .withExpenditures(HelperUtils.calculateTotals(budgets, Budget::getExpenditures));
+      .withExpenditures(HelperUtils.calculateTotals(budgets, Budget::getExpenditures))
+      .withCredits(HelperUtils.calculateTotals(budgets, Budget::getCredits));
     return new LedgerFiscalYearTransactionsHolder(fiscalYearId, ledger, budgets);
   }
 
-  //    #allocated = initialAllocation.add(allocationTo).subtract(allocationFrom)
-  //    #totalFunding = allocated.add(netTransfers)
-  //    #available = totalFunding.subtract(unavailable)
-  //    #cashBalance = totalFunding.subtract(expended)
-  //    #overEncumbered = encumbered.subtract(totalFunding.max(BigDecimal.ZERO)).max(BigDecimal.ZERO)
-  //    #overExpended = expended.add(awaitingPayment).subtract(totalFunding.max(BigDecimal.ZERO)).max(BigDecimal.ZERO)
+  /**
+   * The method follows this formula: <br>
+   * <p>
+   * allocated = initialAllocation + allocationTo - allocationFrom <br>
+   * totalFunding = allocated + netTransfers <br>
+   * available = totalFunding - unavailable <br>
+   * cashBalance = totalFunding - expended + credited <br>
+   * overEncumbered = encumbered - totalFunding <br>
+   * overExpended = expended - credited + awaitingPayment - totalFunding
```

Comment:
same here

Reply:
fixed

Reply:
probably forget to change this place too, fixed